spec/models/journaled/bulk_delivery_spec.rb

```diff
@@ -41,11 +41,7 @@
 
     it 'makes requests to AWS to put the events on the Kinesis with the correct bodies' do
       allow(kinesis_client).to receive(:put_records).and_call_original
-      response = subject.perform
```

Comment:
This was checking the return value of perform which isn't something that can be relied on with DJs so removed it!